helpers/guilds/getBans.ts

```diff
@@ -4,8 +4,14 @@ import { DiscordBan } from "../../types/discord.ts";
 import { User } from "../../transformers/member.ts";
 
 /** Returns a list of ban objects for the users banned from this guild. Requires the BAN_MEMBERS permission. */
-export async function getBans(bot: Bot, guildId: bigint) {
-  const results = await bot.rest.runMethod<DiscordBan[]>(bot.rest, "get", bot.constants.endpoints.GUILD_BANS(guildId));
+export async function getBans(bot: Bot, guildId: bigint, options: GetBans) {
+  const results = await bot.rest.runMethod<DiscordBan[]>(
+    bot.rest,
+    "get",
+    `${bot.constants.endpoints.GUILD_BANS(guildId)}?limit=${options.limit ?? 1000}&before=${
+      options.before ?? null
+    }&after=${options.after ?? null}`,
```

Comment:
@itohatweb @Skillz4Killz is this how I am supposed to do query string params?

Reply:
You can use an if statement

Reply:
alr then